libs/shared/left-side-menu/src/lib/shared-left-side-menu.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+import './shared-left-side-menu.scss';
+
+import {
+  SwipeableDrawer,
+  List,
+  ListItem,
+  Typography,
+  ListItemText
+} from '@material-ui/core';
+
+export interface LeftSideMenuProps {
+  menuItemsArray: Array<Type>
+}
+
+interface Type {
+  name: string;
+  path?: string
+}
+
+export const LeftSideMenu = (props: LeftSideMenuProps) => {
+  const [state, setState] = React.useState({
+    menuOpen: true,
+  });
+
+  const menuItems = props.menuItemsArray;
```

Comment:
Use   
```
useEffect(() => {

  }, [props]); 
```
to load menuItems

Reply:
you can directly use props.menuItemsArray on map, no need reassign to variable.